pipelines/pipelines/buildutils/build-pr.yaml

```diff
@@ -134,9 +152,99 @@ spec:
       value:
         - "--build-arg=dockerRepository=harbor.galasa.dev"
         - "--build-arg=branch=$(params.headSha)"
+    workspaces:
+     - name: git-workspace
+       workspace: git-workspace 
+# 
+# 
+#
+  - name: make-openapi2beans
+    taskRef:
+      name: make-with-params
+    runAfter:
+      - clone-buildutils
+    params:
+    - name: directory
+      value: $(context.pipelineRun.name)/buildutils/openapi2beans
     workspaces:
      - name: git-workspace
        workspace: git-workspace
+#
+#
+#
+  - name: build-openapi2beans-binary-downloadables
+    taskRef:
+      name: docker-build
+    runAfter:
+    - clone-automation
+    - make-openapi2beans
+    params:
+    - name: pipelineRunName
+      value: $(context.pipelineRun.name)
+    - name: context
+      value: $(context.pipelineRun.name)
+    - name: dockerfilePath
+      value: automation/dockerfiles/openapi2beans/openapi2beans-binary-dockerfile
+    - name: imageName
+      value: harbor.galasa.dev/galasadev/openapi2beans-binary-downloadables:$(params.imageTag)
+    - name: noPush
+      value: ""
+    - name: buildArgs
+      value:
+        - "--build-arg=branch=$(params.branch)"
+    workspaces:
+     - name: git-workspace
+       workspace: git-workspace 
+# 
+# 
+# 
+  - name: recycle-openapi2beans-binaries
```

Comment:
don't have to recycle this because the docker image hasn't changed. It will be a no-op.